### ISSUE TITLE
Live Capture - Filter by port?

### ISSUE THREAD

**Issue by thomassross** [desowin/usbpcap#17]
I have this tree:

```
Following filter control devices are available:
1 \\.\USBPcap4
  \??\USB#ROOT_HUB20[snip]
    [Port 1] Generic USB Hub
      [snip]
      [Port 4] USB Composite Device
        USB Input Device
          HID Keyboard Device
        USB Input Device
          HID-compliant device
          HID-compliant consumer control device
[snip]
Select filter to monitor (q to quit):
```

I want to filter only:

```
[Port 4] USB Composite Device
  USB Input Device
    HID Keyboard Device
```

which is in `\\.\USBPcap4`.

Is this possible yet? 
If so, how can I do it with the `-d` option?

**Comment by desowin** [comment]
This is possible using extcap interface which is currently in git and will be available in next official release.

**Comment by zhangshengping** [comment]
How can I filter it on wireshark ? has the feature been add to newest version?

**Comment by desowin** [comment]
Copy USBPcapCMD.exe to c:\Program Files\Wireshark\extcap and restart Wireshark. Then the USBPcap interfaces should be listed in Wireshark GUI. Open configuration window, unselect "Capture from all devices" and "Capture from newly conencted devices", select (in Qt GUI it works via highlighting) the "USB Composite Device" (or the device you want to capture from) and then start capture.

Note that there is a bug in USBPcapCMD which results in USBPcapCMD using virtually whole single core of your CPU (if you have just one core, then it's 100% CPU usage) which will be fixed once the 1.2.0.4 is released. See #32 for more information.
